UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/FileShareServiceController.cs

```diff
@@ -219,11 +221,16 @@ public ActionResult RedirectDownloadFile(string batchId, string fileName)
             byte[] bytes = null;
             if (!string.IsNullOrEmpty(fileName))
             {
-                bytes = fileShareService.GetFileData(configuration["HOME"], batchId, fileName);
+                bytes = fileShareService.GetFileData(configuration["HOME"], Sanitise(batchId), fileName);
                 HttpContext.Response.Headers.Add("x-redirect-status", "true");
             }
 
             return File(bytes, "application/octet-stream", fileName);
         }
+
+        public static string Sanitise(string text)
```

Comment:
This is a very generic method name for a rather specific sanitisation.  Should it be called something like `SanitiseBatchId`?  For example, it wouldn't be correct to use the current method to sanitise any other input.